client/app/components/Input/InputRadio.jsx

```diff
@@ -0,0 +1,47 @@
+// @flow
+import React, { useState } from 'react';
+import css from './InputRadio.scss';
+import type { Option } from './utils';
+
+export type Props = {
+  name?: string,
+  value?: any,
+  options: Option[],
+  onChange?: Function,
+}
+
+export function InputRadio({
+  name,
+  options,
+  onChange,
+  value: propValue,
+}: Props) {
+  const [value, setValue] = useState<any>(propValue);
```

Comment:
I think we can make this uncontrolled instead of controlled. By that I mean, we can rely on the native `<input>` element to update the value for us instead of handling it ourselves in state.

Here's an [article](https://itnext.io/controlled-vs-uncontrolled-components-in-react-5cd13b2075f9) on uncontrolled vs. controlled components if you need it.

Reply:
Thanks for this suggestion! Should I still make this change even though `InputRadio.jsx` represents a collection of radio buttons instead of a single radio button ?

Reply:
Hmm good question. I think you should make that change. InputRadio should just refer to one radio button. We could make some kind of collection component later.

Reply:
Sounds good!, I have changed the name to `InputRadioGroup.jsx` on my latest commit.